examples/chip-tool/commands/clusters/Commands.h

```diff
@@ -10702,6 +10723,88 @@ class ReadOnOffClusterRevision : public ModelCommand
 | * ClusterRevision                                                   | 0xFFFD |
 \*----------------------------------------------------------------------------*/
 
+/*
+ * Command AddOpCert
+ */
+class OperationalCredentialsAddOpCert : public ModelCommand
+{
+public:
+    OperationalCredentialsAddOpCert() : ModelCommand("add-op-cert")
+    {
+        AddArgument("noc", &mNoc);
+        AddArgument("iCACertificate", &mICACertificate);
```

Comment:
`iCACertificate` is very odd casig. Suggest `ICACertificate`.

Reply:
Will provide a fix in a followup. This is because the template is doing `{{asCamelCased name}}` omitting the second argument of https://github.com/project-chip/zap/blob/7c928747ef6c32671d39901c881b82f57a6e1c08/src-electron/generator/helper-c.js#L200 which force the first letter to be lower cased.